docs/azure/sdk/snippets/logging/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <LogWithoutClientRegistration>
+using Azure.Identity;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Azure;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddRazorPages();
+builder.Services.AddAzureClientsCore();
+
+using var logForwarder = builder.Services
+    .BuildServiceProvider()
+    .GetService<AzureEventSourceLogForwarder>();
+logForwarder?.Start();
```

Comment:
```suggestion
    .GetRequiredService<AzureEventSourceLogForwarder>();
logForwarder.Start();
```

Instead of questioning whether the `logForwarder` is `null`, you could use the `GetRequiredService` instead to ensure it is non-null.